src/test/java/com/stripe/functional/TopupTest.java

```diff
@@ -38,6 +38,20 @@ public void testCancel() throws StripeException {
     );
   }
 
+  @Test
+  public void testCancelEmpty() throws StripeException {
+    final Topup topup = getTopupFixture();
+
+    final Topup canceledTopup = topup.cancel();
+
+    assertNotNull(canceledTopup);
+    verifyRequest(
+            ApiResource.RequestMethod.POST,
+            String.format("/v1/topups/%s/cancel", topup.getId()),
+            null
+    );
+  }
+
```

Comment:
You can remove that test since we already test the other methods. We don't do one per prototype since they ultimately call each other!

Reply:
👍 removed!